app/code/Magento/Inventory/Model/ResourceModel/PredefinedId.php

```diff
@@ -28,7 +27,7 @@
      * @param AbstractModel $object
      * @return bool
      */
-    protected function isObjectNotNew(AbstractModel $object)
+    protected function isObjectNotNew(AbstractModel $object): bool
```

Comment:
Why do we need this trait if we already have one
https://github.com/magento-engcom/msi/blob/2.3-develop/lib/internal/Magento/Framework/Model/ResourceModel/PredefinedId.php